tests/contracts/hello/src/lib.rs

```diff
@@ -0,0 +1,191 @@
+#![feature(wasm_abi)]
+
+extern crate alloc;
+
+use oasis_contract_sdk as sdk;
+use oasis_contract_sdk::types::{
+    env::{QueryRequest, QueryResponse},
+    event::Event,
+    message::{Message, NotifyReply, Reply},
+    storage::StoreKind,
+};
+
+pub struct HelloWorld;
+
+#[derive(Clone, Debug, cbor::Encode, cbor::Decode)]
+pub enum Request {
+    #[cbor(rename = "instantiate")]
+    Instantiate,
```

Comment:
is this variant needed/used. I don't see it used in any of the `match request` cases. 

Since this is a contract used as an example, maybe also a simple comment for each of the variants here would be useful.